README.rst

```diff
@@ -11,3 +11,5 @@ These files are:
 also supports a, q, z
 
 gerrit 3
+
+this is a pr testing on gerrit 3
```

Comment:
**mike bayer** (zzzeek) wrote:

comment on the actual line that changed.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542